solution.js

```diff
@@ -1,5 +1,149 @@
 const { nums, words } = require("./data/data.js");
 
+class Node {
+  constructor(data) {
+    this.data = data;
+    this.next = null;
+  }
+}
+
+let newNode = new Node(5);
+let secondNode = new Node(6);
+newNode.next = secondNode;
+
+class LinkedList {
+  constructor(head = null) {
+    this.head = head;
+  }
+
+  insert(value) {
+    const newNode = new Node(value);
```

Comment:
When we insert into a linked list, we insert at the top of it, or in other words, a new node that's inserted should become the new head of the linked list.